### DISCUSSION TITLE
Sensors available

### DISCUSSION THREAD

**Discussion by javico88** [guerrerotook/securitas-direct-new-api#172] [General]
Hi guys,
First of all, thanks a lot for this great integration. 
Is it posible to retrive information about the presence sensors, cameras or door and windows sensors?

**Comment by alruallm** [comment]
Hi All,
In addition to @javico88 question.....

It will be very cool being able to "be aware" if during the "arming request"  at least one "shock door/windows sensors" is open. Now in the current version arming fails, and that's all.

 When you use the Mobile App a pop-up message advices you saying that  the "shock sensor" is open and ask if you want to "ignore and arm" or "cancel de arm request".
